tasks/orka-init.yaml

```diff
@@ -23,6 +42,26 @@ spec:
       type: string
       description: Enables or disables VNC for the VM.
       default: "true"
+    - name: system-serial
```

Comment:
It would probably make more sense to move this to the `orka-deploy` task ... since typically for folks that are using it, you'd want a unique serial per VM